### ISSUE TITLE
Is GOPT designed to take only complete words or sentences? 

### ISSUE THREAD

**Issue by TheoSeo93** [YuanGongND/gopt#5]
Hi again :) Thanks to your response, I was able to replicate the whole process of pronunciation assessment. 
I became curious whether this gopt can provide the accuracy of a phoneme separately without giving a complete word or sentence.
For example, if a speaker speaks out a singular phonetic sound "AH" and provide the correct phonetic transcription "AH" within the gop kaldi recipe, will it still generate accuracy of the phoneme? 
I wasn't so sure if this is designed to take only complete words or sentences, or it can take singular phoneme separately.

Thank you very much
Best regards
Theo Seo

**Comment by YuanGongND** [comment]
Yes - I think the reason why GOPT output the baseline is mainly because it takes context into consideration, i.e., it takes input sequence longer than a single phone. The input sequence, however, doesn't have to be a full sentence, it can be a word, a phrase.

If you are interested in context-independent single-phone classification, that is our baseline, and is implemented in the original Kaldi/gop recipe, which I believe is an implementation of the following paper.

Hu, W., Qian, Y., Soong, F. K., & Wang, Y. (2015). Improved mispronunciation detection with deep neural network trained acoustic models and transfer learning based logistic regression classifiers. Speech Communication, 67(January), 154-166.

-Yuan
